arch/arm/src/stm32/hardware/stm32g47xxx_pwr.h

```diff
@@ -0,0 +1,485 @@
+#define PWR_CR4_WP1                    (0x1 << PWR_CR4_WP1_SHIFT)
+
+/* PWR Power Status Register 1 (SR1) */
+
+#define PWR_SR1_WUFI_SHIFT             (15)                               /* Wake Up Flag Internal */
+#define PWR_SR1_WUFI                   (0x1 << PWR_SR1_WUFI_SHIFT)
+#define PWR_SR1_SBF_SHIFT              (8)                                /* Stand-By Flag */
+#define PWR_SR1_SBF                    (0x1 << PWR_SR1_SBF_SHIFT)
+#define PWR_SR1_WUF_SHIFT              (0)                                /* Wake Up Flags */
+#define PWR_SR1_WUF_MASK               (0x1f << PWR_SR1_WUF_SHIFT)
+#  define PWR_SR1_WUF5                 (0x10 << PWR_SR1_WUF_SHIFT)        /* Wake Up Flag 5 */
+#  define PWR_SR1_WUF4                 (0x8 << PWR_SR1_WUF_SHIFT)         /* Wake Up Flag 4 */
+#  define PWR_SR1_WUF3                 (0x4 << PWR_SR1_WUF_SHIFT)         /* Wake Up Flag 3 */
+#  define PWR_SR1_WUF2                 (0x2 << PWR_SR1_WUF_SHIFT)         /* Wake Up Flag 2 */
+#  define PWR_SR1_WUF1                 (0x1 << PWR_SR1_WUF_SHIFT)         /* Wake Up Flag 1 */
+
+/* PWR Power Status Register 2 (SR2) */
+
+#define PWR_SR2_PVMO4_SHIFT            (15)                               /* Peripheral Voltage Monitoring Output 4 */
+#define PWR_SR2_PVMO4                  (0x1 << PWR_SR2_PVMO4_SHIFT)
+#define PWR_SR2_PVMO3_SHIFT            (14)                               /* Peripheral Voltage Monitoring Output 3 */
+#define PWR_SR2_PVMO3                  (0x1 << PWR_SR2_PVMO3_SHIFT)
+#define PWR_SR2_PVMO2_SHIFT            (13)                               /* Peripheral Voltage Monitoring Output 2 */
+#define PWR_SR2_PVMO2                  (0x1 << PWR_SR2_PVMO2_SHIFT)
+#define PWR_SR2_PVMO1_SHIFT            (12)                               /* Peripheral Voltage Monitoring Output 1 */
+#define PWR_SR2_PVMO1                  (0x1 << PWR_SR2_PVMO1_SHIFT)
+#define PWR_SR2_PVDO_SHIFT             (11)                               /* Power Voltage Detector Output */
+#define PWR_SR2_PVDO                   (0x1 << PWR_SR2_PVDO_SHIFT)
+#define PWR_SR2_VOSF_SHIFT             (10)                               /* Voltage Scaling Flag */
+#define PWR_SR2_VOSF                   (0x1 << PWR_SR2_VOSF_SHIFT)
+#define PWR_SR2_REGLPF_SHIFT           (9)                                /* Low-power Regulator Flag */
+#define PWR_SR2_REGLPF                 (0x1 << PWR_SR2_REGLPF_SHIFT)
+#define PWR_SR2_REGLPS_SHIFT           (8)                                /* Low-power Regulator Started */
+#define PWR_SR2_REGLPS                 (0x1 << PWR_SR2_REGLPS_SHIFT)
+#define PWR_SR2_FLASHRDY_SHIFT         (7)                                /* Flash Ready Flag */
+#define PWR_SR2_FLASHRDY               (0x1 << PWR_SR2_FLASHRDY_SHIFT)
+
+/* PWR Power Status Reset Register (SCR) */
+
+#define PWR_SCR_CSBF_SHIFT      (8)                                       /* Clear Standby Flag */
```

Comment:
line then up please

Reply:
Fixed in 076cceb7b7da7510a6c7aed40cae7789343956c0.